### PR TITLE
docs(meshservice): update BackendRefs to use real resources

### PR DESCRIPTION
**Pull request by lobkovilya**
Fix https://github.com/kumahq/kuma-website/issues/1998

### CODE REVIEW

File: app/_src/policies/meshhttproute.md [hunk 1]
```diff
@@ -450,13 +500,13 @@ spec:
                     tags:
```
Comment (michaelbeaumont) [inline app/_src/policies/meshhttproute.md:500]: Does the mirror not need to be updated?

File: app/_src/using-mesh/managing-ingress-traffic/builtin-routes.md [hunk 2]
```diff
@@ -42,6 +43,39 @@ spec:
...
+```yaml
+type: MeshHTTPRoute
+name: edge-gateway-route
+mesh: default
+spec:
+  targetRef:
+    kind: MeshGateway
+    name: edge-gateway
+    tags: # optional, for selecting specific listeners
+      port: http/8080
```
Comment (michaelbeaumont) [inline app/_src/using-mesh/managing-ingress-traffic/builtin-routes.md:59]:
```suggestion
      port: http-8080
```

File: app/_src/policies/meshhttproute.md (hunk 1 above)
Comment (lobkovilya) [reply app/_src/policies/meshhttproute.md:500]: Unfortunately not yet https://github.com/kumahq/kuma/issues/11341

File: app/_src/policies/meshhttproute.md [hunk 3]
```diff
@@ -373,26 +421,24 @@ spec:
...
+type: MeshHTTPRoute
+name: http-route-1
+mesh: default
 spec:
   targetRef:
-    kind: MeshService
-    name: frontend_kuma-demo_svc_8080
+    kind: MeshSubset
+    tags:
+      app: frontend
```
Comment (lobkovilya) [inline app/_src/policies/meshhttproute.md:441]:
```suggestion
      kuma.io/service: frontend_kuma-demo_svc_8080
```

File: app/_src/policies/meshhttproute.md [hunk 4]
```diff
@@ -419,16 +465,20 @@ spec:
...
 type: MeshHTTPRoute
 name: http-route-1
 mesh: default
 spec:
   targetRef:
-    kind: MeshService
-    name: frontend_kuma-demo_svc_8080
+    kind: MeshSubset
+    tags:
+      app: frontend
```
Comment (lobkovilya) [inline app/_src/policies/meshhttproute.md:481]:
```suggestion
      kuma.io/service: frontend_kuma-demo_svc_8080
```

File: app/_src/policies/meshtcproute.md [hunk 5]
```diff
@@ -102,25 +103,20 @@ different tags and implement A/B testing or canary deployments.
...
   targetRef:
     kind: MeshService
     name: frontend_kuma-demo_svc_8080
   to:
     - targetRef:
-        kind: MeshService
-        name: backend_kuma-demo_svc_3001
+        kind: MeshSubset
+        tags:
+          app: backend
```
Comment (lobkovilya) [inline app/_src/policies/meshtcproute.md:119]:
```suggestion
          kuma.io/service: backend_kuma-demo_svc_3001
```

File: app/_src/policies/meshtcproute.md [hunk 6]
```diff
@@ -151,27 +153,23 @@ spec:
...
```
Comment (lobkovilya) [inline app/_src/policies/meshtcproute.md:158]:
```suggestion
          kuma.io/service: backend_kuma-demo_svc_3001
```

File: app/_src/policies/meshhttproute.md [hunk 7]
```diff
@@ -302,6 +304,52 @@ spec:
...
+{% endif_version %}
+
+{% if_version gte:2.9.x %}
+
+### Traffic split
+
+We can use `MeshHTTPRoute` to split an HTTP traffic between different MeshServices
+implementing A/B testing or canary deployments. 
+If we want to split traffic between `v1` and `v2` versions of the same service,
+first we have to create MeshServices `backend-v1` and `backend-v2` that select
```
Comment (Automaat) [inline app/_src/policies/meshhttproute.md:316]: do we have docs on how to manually create MeshService? Can we link it here?

Comment (lobkovilya) [reply app/_src/policies/meshhttproute.md:265]: AFAIK there is only https://kuma.io/docs/2.9.x/networking/meshservice/, do you think we should link it?

Comment (lobkovilya) [reply app/_src/policies/meshhttproute.md:265]: created an issue https://github.com/kumahq/kuma-website/issues/2028

**Review by Automaat — APPROVED** [review]

**Pull request merged**